app/templates/index.hbs

```diff
@@ -44,7 +46,11 @@
                 <div class="col-xs-12">
                     <h2>{{t "index.subjects.heading"}}</h2>
                     <div class="subjectsList p-lg"> {{!SUBJECTS LIST}}
-                        {{taxonomy-top-list list=model.taxonomies}}
+                        {{#if theme.provider.additionalProviders}}
+                            {{additional-provider-list additionalProviders=theme.provider.additionalProviders}}
```

Comment:
I thought the presence of additionalProviders made more sense here, because if they exist, then that is what we show.  Otherwise, we show taxonomies.

I have expressed that these should be temporary changes to get ASU out the door. (You can see some of my comments in the JIRA ticket for this).   There will be a lot of usecases that this PR doesn't adequately handle, and many more options that other collections might want. ASU is being treated like a branded preprint provider here, but it is actually a collection of lots of resource types, not just preprints, and is not a single brand, but a bunch of providers.  

I am uncomfortable putting a bunch of special case handling in these templates for one type of collection and think these pages need to become more API-driven than they already are.